[Bug]: window.Livewire.restart() is not a function

### What did you expect? 🧐

Undefined methods should not be called 

### What actually happened? 😵‍💫

![image](https://github.com/swup/livewire-plugin/assets/1425304/39696743-bf32-4c72-be68-e6e85f2e13ad)
This function being called does not exist

### Swup and plugin versions 📝

- swup: 4.6.1
- livewire-plugin: 2.0.0


### What browsers are you seeing the problem on? 🧭

_No response_

### Relevant log output 🤓

```shell
Hooks.ts:440 Error in hook 'content:replace': TypeError: window.Livewire.restart is not a function
    at i.refreshLivewireComponents (index.js:47:19)
    at index.ts:38:27
    at new Promise (<anonymous>)
    at f (index.ts:37:13)
    at S.run (Hooks.ts:434:26)
    at S.call (Hooks.ts:358:14)
    at async j.T [as renderPage] (renderPage.ts:21:23)
    at async navigate.ts:196:6
run @ Hooks.ts:440
await in run (async)
call @ Hooks.ts:358
await in call (async)
T @ renderPage.ts:25
(anonymous) @ navigate.ts:196
```


### URL to minimal reproduction 🔗

This would require a hosted Laravel livewire app

### Checked all these? 📚

- [X] I have read and searched the [official docs](https://swup.js.org)
- [X] I have checked [discussions](https://github.com/orgs/swup/discussions) and [existing issues](https://github.com/issues?q=is:issue+org:swup) for related problems
- [X] I agree to follow this project's [Code of Conduct](https://github.com/swup/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have provided all necessary information to the best of my knowledge

--- a/src/index.js
+++ b/src/index.js
@@ -43,7 +43,21 @@ export default class SwupLivewirePlugin extends Plugin {
 			el.setAttribute('wire:initial-data', JSON.stringify(data));
 		});
 
-		// Aleays restart Livewire
-		window.Livewire.restart();
+		// Always restart Livewire
+		if (typeof window.Livewire.restart === 'function') {
+			window.Livewire.restart();
+		} else {
+			// handle versions of Livewire that don't have the .restart() method
+			window.Livewire.all().map((component) => {
+				// not always defined - depends on the component
+				if (typeof component.call === 'function') {
+					return component.call('restart');
+				}
+				// alternative version to component.call('restart') in case that doesn't exist
+				if (typeof component.$wire?.$refresh === 'function') {
+					component.$wire.$refresh();
+				}
+			});
+		}
 	}
 }

